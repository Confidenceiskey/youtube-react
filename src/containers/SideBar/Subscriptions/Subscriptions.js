import React, { Component } from 'react';
import { Divider } from 'semantic-ui-react';
import SideBarHeader from '../SideBarHeader/SideBarHeader';
import Subscription from './Subscription/Subscription';

class Subscriptions extends Component {
  render() {
    return (
      <React.Fragment>
        <SideBarHeader title='Subscriptions' />
        <Subscription label='Music Channel' broadcasting />
        <Subscription label='Coursera' amountNewVideos={10} />
        <Subscription label='TEDx Talks' amountNewVideos={23}/>
        <Subscription label='Standford iOs' amountNewVideos={4} />
        <Subscription label='Udacity' amountNewVideos={114} />
        <Divider />
      </React.Fragment>
    );
  }
}

export default Subscriptions;
