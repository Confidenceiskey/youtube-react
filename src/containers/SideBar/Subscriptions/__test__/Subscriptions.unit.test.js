import React from 'react';
import { shallow } from 'enzyme';
import Subscriptions from '../Subscriptions';

it('renders Subscriptions', () => {
  const wrapper = (
    <Subscriptions />
  );
  expect(wrapper).toMatchSnapshot();
});
