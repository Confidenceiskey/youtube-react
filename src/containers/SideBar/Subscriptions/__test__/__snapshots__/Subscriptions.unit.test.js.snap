// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Subscriptions 1`] = `<Subscriptions />`;
